x-pack/plugins/security_solution/docs/openapi/ess/security_solution_timeline_api_2023_10_31.bundled.schema.yaml

```diff
@@ -131,7 +131,7 @@ paths:
                   type: string
               required:
                 - note
-        description: 'The note to add or update, along with additional metadata.'
```

Comment:
@rbrtj I wonder what triggered these changes?

Reply:
Honestly, I have no idea 🤔 
But they are gone after updating the branch with the main